utils.js

```diff
@@ -136,3 +142,13 @@ function removeStylesheets(win) {
   }
 }
 exports.removeStylesheets = removeStylesheets;
+
+function installStylesheet(win, uri) {
```

Comment:
sure, it doesn't really remove duplication. its the same amount of text. but one fn now has to jump to another before actually doing anything